libs/application/templates/new-primary-school/src/forms/NewPrimarySchoolForm/childrenNParentsSection/childInfoSubSection.ts

```diff
@@ -76,25 +76,44 @@ export const childInfoSubSection = buildSubSection({
             getApplicationExternalData(application.externalData).applicantCity,
         }),
         buildTextField({
-          id: 'childInfo.chosenName',
-          title: newPrimarySchoolMessages.childrenNParents.childInfoChosenName,
+          id: 'childInfo.preferredName',
+          title:
+            newPrimarySchoolMessages.childrenNParents.childInfoPreferredName,
           width: 'half',
           defaultValue: (application: Application) =>
             getApplicationExternalData(application.externalData)
               .childInformation.preferredName,
```

Comment:
Make `undefined` if `preferredName` is `null`, to remove warning in console and so we don't save the value in answers
> Warning: `value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.

```suggestion
          defaultValue: (application: Application) =>
            getApplicationExternalData(application.externalData)
              .childInformation.preferredName ?? undefined,
```